dotcom-rendering/src/layouts/FrontLayout.tsx

```diff
@@ -330,13 +311,20 @@ export const FrontLayout = ({ front, NAV }: Props) => {
 					)}
 				</>
 			</div>
-
 			<main
 				data-layout="FrontLayout"
 				data-link-name={`Front | /${front.pressedPage.id}`}
 				id="maincontent"
 				css={hasPageSkin && pageSkinContainer}
 			>
+				{isNetworkFrontPageId(pageId) && (
+					<Island priority="enhancement" defer={{ until: 'idle' }}>
```

Comment:
Do you think this would qualify as a feature?

Reply:
I wanted to give it the same priority as [the subnav](https://github.com/guardian/dotcom-rendering/blob/doml/edition-switcher-banner-with-islands/dotcom-rendering/src/layouts/FrontLayout.tsx#L283), to avoid potential rendering issues. I don't _think_ it would cause rendering issue due to one component loading before the other. Do you think it's worth making it a feature? I agree that "feature" is more accurate than "enhancement" here.

Reply:
Good point about consistency, maybe one for a future fix…